app/components/publish-actions.tsx

```diff
@@ -161,20 +168,22 @@ const Embed = ({ chartWrapperRef, configKey, locale }: PublishActionProps) => {
                   "For embedding visualizations in systems without JavaScript support (e.g. WordPress).",
               })}
             />
-            <EmbedToggleSwitch
-              value="remove-border"
-              checked={isWithoutBorder}
-              onChange={handleStylingChange}
-              label={t({
-                id: "publication.embed.iframe.remove-border",
-                message: "Remove border",
-              })}
-              infoMessage={t({
-                id: "publication.embed.iframe.remove-border.warn",
-                message:
-                  "For embedding visualizations in systems without a border.",
-              })}
-            />
+            {state?.chartConfigs.length === 1 && (
```

Comment:
Nit: This is an edge case, but technically we can:
- have two charts in the editing mode,
- proceed to layout options,
- switch to a dashboard layout (free canvas),
- remove a chart using built-in, more options menu,
- publish.

See that we have a dashboard with only one chart; from what I understood, we should not be able to remove a border in this case, but only for single-tab-layout charts? Let me know if this makes sense, if so, I'd also add a `state.layout.type === "tab"` to the condition (see https://visualization-tool-git-feat-border-embed-layouts-ixt1.vercel.app/en/v/rVdrzb3s6Tmm?dataSource=Prod) 👀